opentracing/src/main/java/opentracing/Span.java

```diff
@@ -25,16 +25,7 @@
   /**
    * Suitable for serializing over the wire, etc.
    */
-  TraceContext traceContext();
-
-  /**
-   * Denotes the beginning of a subordinate unit of work.
-   *
-   * @param operationName name of the operation represened by the new span from the perspective of
-   * the current service.
-   * @return a new child Span in "started" state.
-   */
-  Span startChild(String operationName);
+  SpanContext childContext();
```

Comment:
>  there must be a way to retrieve span's own context, so that it can be serialized on the wire.

That makes more sense to me.

One creates the initial SpanContext from the `TraceContext.newRoot()`.
From a SpanContext one creates child Spans using `Tracer.joinTrace(..)` or `Tracer.startSpanWithContext(..)`.

So if we rename the method instead to `Span.context()` (since the original `traceContext` is now inaccurate and `spanContext` superfluous), when you add tags to a SpanContext it's either when you have
 - the root SpanContext (from `TraceContext.newRoot()`), or
 - a current Span, and can add tags to its SpanContext (from `Span.context()`)¹

¹ This is not entirely intuitive because you're adding tags to the Span's context. That Span isn't going to take on these tags (because it's already been constructed) but further children of the Span will take on these tags).

As far as exposing the Span's current context for the sake of serialisation I don't (yet) see the need for that for that same reasons as I don't see the need for `TraceCodec`.

> The specification explains the need for serialisation.
> But I don't (yet) see the need for the API to impose how it should be done.